game/src/ungap/layers.rs

```diff
@@ -363,33 +362,18 @@ fn make_zoom_controls(ctx: &mut EventCtx) -> Widget {
     ])
 }
 
-fn legend_batch(ctx: &mut EventCtx, color: Color, txt: Text) -> GeomBatch {
-    // TODO Height of the "trail" button is slightly too low!
-    // Text with padding and a background color
-    let (mut batch, hitbox) = txt
-        .render(ctx)
-        .batch()
-        .container()
+fn legend_btn(color: Color, label: &str) -> ButtonBuilder {
```

Comment:
This is a testament to the nice ButtonBuilder API and the polish there -- next time I'll copy from more modern examples. :)